portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,31 +3,87 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
 
 public class Comment {
```

Comment:
this class design is a little confusing. I would recommend making this class "CommentUtil" or something, since it is not going to be instantiated and only has static helper methods. Then put the Comment class as either a private static nested class, since it looks like it is never instantiated outside of this file.

Reply:
Yes. That would make more sense. I will try to make a separate class for Response that inherits Comment class and probably make them both private.

Reply:
Alright, I refactored this into 3 classes. Comment, Response and CommentUtil. CommentUtil manipulates Comment and Response objects.